### ISSUE TITLE
Play song suggestions error 

### ISSUE THREAD

**Issue by heyjumanji** [SudhanPlayz/Discord-MusicBot#1202]
V5 the latest update where the auto leave command was added after that the play command song suggestions not even loading also the auto leave not make sense as we have 247 cmd as the both cmd do the same i report to remove the cmd and fix the issue

**Comment by Assassin654** [comment]
I have no issues. The auto leave does not interact with the play command so it is a different issue you are experiencing. Be sure you have all the libraries installed/updated as the play song suggestions has been updated and now uses a library instead of the lavalink. A recent update to the discord API has also broken some things, so be sure you also have the latest version of that. 

Also the auto leave is not the same as 24/7 

The biggest feature of the command allows the bot to be in the vc without constantly disconnecting and reconnecting after songs have played, while also not being in 24/7 even after everyone has left the vc.

**Comment by heyjumanji** [comment]
So how do i can use the library instead of lavalink and can you explain the what changes have been done in the new discord api and what things are broken, also when i type /play cmd the bot dont show the suggestion as it should and then we use to select a song to play from the suggestion selection when we type play, also how do i can now use library instead of lavalink. Also i have installed the latest github v5 repo and i got that problem i hope you help me with the situation. Thanks for reading and i hope i get the solution sooner.

**Comment by ItJustFox** [comment]
Actually I got the same problem, on my personnal node, and even on public node ?
![image](https://user-images.githubusercontent.com/42358087/230772195-68d67d7d-dccc-4467-95ef-27e375e788eb.png)
![image](https://user-images.githubusercontent.com/42358087/230772198-c6e35ad9-8ef3-4b6c-ac91-9b7645ae76e9.png)

**Comment by heyjumanji** [comment]
Yeah the play command is broken from the last updated it breaks the whole entire bot

**Comment by Assassin654** [comment]
> Actually I got the same problem, on my personnal node, and even on public node ? ![image](https://user-images.githubusercontent.com/42358087/230772195-68d67d7d-dccc-4467-95ef-27e375e788eb.png) ![image](https://user-images.githubusercontent.com/42358087/230772198-c6e35ad9-8ef3-4b6c-ac91-9b7645ae76e9.png)

is there any errors? the lava link has nothing to do with the look up anymore btw

**Comment by ItJustFox** [comment]
> > Actually I got the same problem, on my personnal node, and even on public node ? ![image](https://user-images.githubusercontent.com/42358087/230772195-68d67d7d-dccc-4467-95ef-27e375e788eb.png) ![image](https://user-images.githubusercontent.com/42358087/230772198-c6e35ad9-8ef3-4b6c-ac91-9b7645ae76e9.png)
> 
> is there any errors? the lava link has nothing to do with the look up anymore btw

There is actually no error on console / logs, there is just no options loading :/

**Comment by LewdHuTao** [comment]
![image](https://user-images.githubusercontent.com/90232327/231927234-c981960e-1346-4374-8e36-d63601501970.png)

im pretty sure the autoleave update has nothing to do with song suggestion as @Assassin654 never touch interactionCreate.js file and play.js file.

**Comment by LewdHuTao** [comment]
![image](https://user-images.githubusercontent.com/90232327/231927583-72bb1d12-9f88-430f-9beb-6127898b1d4d.png)

they works fine lol (yes this is the latest version), and tested with public lavalink

**Comment by LewdHuTao** [comment]
TL;DR

* Reinstall again the bot
* Make sure you and your bot have good internet connection

i'll close this, feel free to reopen if this issue still occured
